Blog-projects/Widget-window-in-Qt-Quick-app/fontsbackend.cpp

```diff
@@ -0,0 +1,13 @@
+#include "fontsbackend.h"
+
+#include <QFontDatabase>
+
+FontsBackend::FontsBackend(QObject *parent)
+    : QObject{parent}
+{
+}
+
+const QStringList FontsBackend::fontFamilyList()
```

Comment:
const at the end

Reply:
Static member function cannot have const qualifier. See fontsbackend.h#14

Reply:
Removed const instead.

Reply:
Ah, right, yes.